lua/ulx/modules/sh/timed_gag.lua

```diff
@@ -0,0 +1,98 @@
+CATEGORY_NAME = "CFC"
+SQL_TABLE = "cfc_timed_gags"a
+
+local GaggedPlayers = {}
+
+local function createTable()
+    if sql.TableExists( SQL_TABLE ) then return end
+
+    local createTableQuery = string.format("CREATE TABLE %s(steam_id TEXT, expiration BIGINT)", SQL_TABLE)
+
+    sql.Query(createTableQuery)
+end
+
+local function initializeGaggedPlayers()
+    local query = string.format( "SELECT * FROM %s", SQL_TABLE)
+
+    GaggedPlayers = sql.Query(query)
+end
+
+
+local function init()
+    createTable()
+
+    initializeGaggedPlayers()
+end
+
+
+local function removeExpiredGag(steamId)
+    GaggedPlayers[steamId] = nil
+
+    local query = string.format("REMOVE FROM %s WHERE steam_id='%s'", SQL_TABLE, steamID)
+    sql.Query(query)
+end
+
+local function playerIsAlreadyGagged(steamId)
```

Comment:
Maybe unimportant but it could be important to distinguish between regular gag and timed gag?